Screenbox/App.xaml

```diff
@@ -18,10 +18,10 @@
                     <!--  Brushes  -->
                     <ResourceDictionary.ThemeDictionaries>
                         <ResourceDictionary x:Key="Default">
-                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorLight2" />
+                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorLight3" />
                         </ResourceDictionary>
                         <ResourceDictionary x:Key="Light">
-                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorDark1" />
+                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorDark2" />
```

Comment:
This resource only applies to Animated icons and thus causes color mismatch

![image](https://github.com/user-attachments/assets/33459144-ee66-4861-ac0d-e99a20b63d11)

Reply:
I will be reverting this so I can merge this PR. Please feel free to submit this change again in another PR.